x/bank/types/genesis_test.go

```diff
@@ -1,52 +1,150 @@
-package types_test
+package types
```

Comment:
```suggestion
package types
```

I know why you're doing this - you need to access private methods. In this case, do you mind renaming this file to `genesis_internal_test.go` please? ref https://github.com/cosmos/cosmos-sdk/pull/8402#discussion_r561264393